server/repository/src/migrations/v2_02_00/remove_changelog_triggers.rs

```diff
@@ -0,0 +1,118 @@
+use crate::migrations::*;
+
+#[cfg(not(feature = "postgres"))]
+pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    sql!(
+        connection,
+        r#"
+        DROP TRIGGER IF EXISTS location__insert_trigger;
```

Comment:
looks like double underscore here.

Reply:
![image](https://github.com/user-attachments/assets/936db7b0-4492-450c-9f26-afc04d69095d)
I think it's correct right?

Reply:
Thanks !

I ran both sqlite and postgres, and looked at schema, 0 triggers.

I did see some artifact trigger methods though, maybe worth deleting, but not critical.

Reply:
![Screenshot 2024-07-22 at 12 35 07 PM](https://github.com/user-attachments/assets/81631e51-6419-4b04-a394-ebde785b6cee)

Reply:
Thanks Andrei had been meaning to come back to that, but forgot thanks for the reminder.
I think it's probably worth cleaning up, as could cause confusion in future...